modules/charon-core/pom.xml

```diff
@@ -64,6 +64,22 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Add `test` scope for these dependencies.

Reply:
fixed in f6c70ca0d7276c5e56af08faad3962e0bb9df4ad